plugins/ipqualityscore/plugin.spec.yaml

```diff
@@ -0,0 +1,795 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: ipqualityscore
+title: IPQS Fraud and Risk Scoring
+description: IPQS Fraud and Risk Scoring plugin provides enterprise grade fraud prevention, risk analysis, and threat detection Analyze IP Addresses, Email Addresses, Phones, URLs and Domains to identify sophisticated bad actors and high risk behavior
+version: 1.0.0
+supported_versions: ["1.0.0"]
+vendor: ipqs
```

Comment:
@joneill-r7 , can you confirm with Holly but I think the vendor may need updated to 'ipqualityscore' and the support to 'community'?